homeassistant/components/aquostv/media_player.py

```diff
@@ -99,7 +99,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     host = config[CONF_HOST]
     remote = sharp_aquos_rc.TV(host, port, username, password, 15, 1)
 
-    add_entities([SharpAquosTVDevice(name, remote, power_on_enabled)])
+    add_entities([SharpAquosTVDevice(name, remote, power_on_enabled)], True)
```

Comment:
Why is this changed, this has nothing to do with using entity class attributes?

Reply:
Since this removes defining the state in the `__init__` block, don't we have to update the entities on add so we avoid undefined states?

Reply:
It was previously initialized with `self._state = None`, `None` is the default for `self._attr_state` so nothing changes.